### ISSUE TITLE
Problem to retrieve action

### ISSUE THREAD

**Issue by ElRochito** [CodeDredd/laravel-soap#133]
Since I updated the package to **v2.0.0-beta.9**, I've got a problem to retrieve SOAP request action.

This method parse header to get SOAPAction but with version **v2.0.0-beta.9**, **SOAPAction** doesn't exists on header, we can see it in the **Content-type** header ...
```
public function action(): string
{
   return $this->request->getHeaderLine('SOAPAction');
}
```

Request dump :
```
...
"Content-Type" => array:1 [
    0 => "application/soap+xml; charset="utf-8"; action="addReservation""
]
...
```

In the **v2.0.0-beta.7** I don't have this problem

Thanks for your answers

**Comment by ElRochito** [comment]
I'm on SOAP 1_2

**Comment by CodeDredd** [comment]
@ElRochito Thanks for the report. I try to look into it.

**Comment by ElRochito** [comment]
Thanks for your answer.

I use SOAPAction for my unit test indeed.

But, when i'm on **SOAP 1_2** with **v2.0.0-beta.7**, it's works. I see **SOAPAction** on the header

**Comment by CodeDredd** [comment]
Ah ok....yeah i fixed the header because they were wrong for 1_2 .... i will make the unit testing work again with SOAP1_2. Because in beta.7 the fake library was using SOAP1_1 even for SOAP1_2 ;-) ... keeping you update.
Thanks for the report!

**Comment by CodeDredd** [comment]
@ElRochito closed with #136
